python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ntp.schema.yml

```
@@ -105,3 +105,45 @@ keys:
         description: List of trusted-keys as string ex. 10-12,15.
         convert_types:
           - int
+      serve:
+        description: Enable Serving NTP to clients
+        type: dict
+        keys:
+          all:
+            type: dict
+            keys:
+              enabled:
+                type: bool
+              vrfs:
+                type: list
+                items:
+                  type: dict
+                  keys:
+                    name:
+                      type: str
+          access_groups:
+            type: list
+            description: Standard ACL to apply to NTP serve. Only one ACL per VRF can be configured. All ACLs are applied in the 'in' direction.
+            items:
```

We would not want a VRF more than once.
```suggestion
            unique_keys:
              - vrf
            items:
```